Sources/Orbit/Support/Forms/InputState.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+public enum InputState {
```

Comment:
No documentation here.

Reply:
Added comment only to the enum, because it will probably be completely removed in future story (and replaced with native swiftui `disabled` and `focusable` modifiers).